ergo/metaculus.py

```diff
@@ -1272,6 +1272,11 @@ def get_questions_for_pages(
 
         questions = get_questions_for_pages(query_string, pages)
 
+        # add additional fields ommited from previous query
+        for i, _q in enumerate(questions):
```

Comment:
Why the underscore variable names?

Reply:
I'll just remove them if you don't like them stylistically. I am used to them, particularly for a single letter variable, as a kind of weak private/ indication that this is a transient variable. Why did I not use it for the `i`? Good question. Seems like I should at least be consistent, but somehow indexes of the resulting object are less transient than the _q which will be mutated and forgotten.